samples/algorithms/SuperdenseCoding.qs

```diff
@@ -9,37 +9,39 @@
 /// This Q# program implements superdense coding to send two classical bits of
 /// information.
 namespace Sample {
+    open Microsoft.Quantum.Measurement;
     open Microsoft.Quantum.Diagnostics;
     open Microsoft.Quantum.Random;
```

Comment:
I just removed all 3 namespaces locally and it still worked fine. I believe Measurement is automatically included, and there are no Diagnostics or Random calls with these changes.